code/modules/mod/modules/modules_maint.dm

```diff
@@ -0,0 +1,247 @@
+		although who knows with the rumors..."
+	icon_state = "tanning"
+	module_type = MODULE_USABLE
+	complexity = 1
+	use_power_cost = DEFAULT_CELL_DRAIN * 5
+	incompatible_modules = list(/obj/item/mod/module/tanner)
+	cooldown_time = 30 SECONDS
+
+/obj/item/mod/module/tanner/on_use()
+	. = ..()
+	if(!.)
+		return
+	playsound(src, 'sound/machines/microwave/microwave-end.ogg', 50, TRUE)
+	var/datum/reagents/holder = new()
+	holder.add_reagent(/datum/reagent/spraytan, 10)
+	holder.trans_to(mod.wearer, 10, methods = VAPOR)
+	if(prob(5))
+		SSradiation.irradiate(mod.wearer)
+	drain_power(use_power_cost)
+
+///Balloon Blower - Blows a balloon.
+/obj/item/mod/module/balloon
+	name = "MOD balloon blower module"
+	desc = "A strange module invented years ago by some ingenious mimes. It blows balloons."
+	icon_state = "bloon"
+	module_type = MODULE_USABLE
+	complexity = 1
+	use_power_cost = DEFAULT_CELL_DRAIN*0.5
+	incompatible_modules = list(/obj/item/mod/module/balloon)
+	cooldown_time = 15 SECONDS
+
+/obj/item/mod/module/balloon/on_use()
+	. = ..()
+	if(!.)
+		return
+	if(!do_after(mod.wearer, 10 SECONDS, target = mod))
+		return FALSE
+	mod.wearer.adjustOxyLoss(20)
+	playsound(src, 'sound/items/inflate_bloon.ogg', 50, TRUE)
+	var/obj/item/toy/balloon/balloon = new(get_turf(src))
+	mod.wearer.put_in_hands(balloon)
+	drain_power(use_power_cost)
+
+///Atrocinator - Flips your gravity.
+/obj/item/mod/module/atrocinator
+	name = "MOD atrocinator module"
+	desc = "A mysterious orb that has mysterious effects when inserted in a MODsuit."
+	icon_state = "atrocinator"
+	module_type = MODULE_TOGGLE
+	complexity = 2
+	active_power_cost = DEFAULT_CELL_DRAIN
+	incompatible_modules = list(/obj/item/mod/module/atrocinator, /obj/item/mod/module/magboot)
+	cooldown_time = 0.5 SECONDS
+	overlay_state_inactive = "module_atrocinator"
+	var/step_count = 0
+	var/you_fucked_up = FALSE
+
+/obj/item/mod/module/atrocinator/on_activation()
+	. = ..()
+	if(!.)
+		return
+	playsound(src, 'sound/effects/curseattack.ogg', 50)
+	mod.wearer.AddElement(/datum/element/forced_gravity, NEGATIVE_GRAVITY)
+	RegisterSignal(mod.wearer, COMSIG_MOVABLE_MOVED, .proc/check_upstairs)
+	mod.wearer.update_gravity(mod.wearer.has_gravity())
+	ADD_TRAIT(mod.wearer, TRAIT_SILENT_FOOTSTEPS, MOD_TRAIT)
+	check_upstairs() //todo at some point flip your screen around
+
+/obj/item/mod/module/atrocinator/on_deactivation()
+	if(you_fucked_up)
+		to_chat(mod.wearer, span_danger("It's too late."))
+		return FALSE
+	. = ..()
+	if(!.)
+		return
+	playsound(src, 'sound/effects/curseattack.ogg', 50)
+	qdel(mod.wearer.RemoveElement(/datum/element/forced_gravity, NEGATIVE_GRAVITY))
+	UnregisterSignal(mod.wearer, COMSIG_MOVABLE_MOVED)
+	step_count = 0
+	mod.wearer.update_gravity(mod.wearer.has_gravity())
+	REMOVE_TRAIT(mod.wearer, TRAIT_SILENT_FOOTSTEPS, MOD_TRAIT)
+	var/turf/open/openspace/current_turf = get_turf(mod.wearer)
+	if(istype(current_turf))
```

Comment:
there are two different kinds of openspace turfs now so this wont work. why check for openspace anyways? zFall() shouldnt do anything if you cant fall

Reply:
the other openspace is space, so it has no gravity, so it doesnt matter